packages/legend-graph/src/helpers/MappingResolutionHelper.ts

```diff
@@ -42,12 +42,14 @@ export const updateRootSetImplementationOnCreate = (
     setImp.class.value,
   ).filter((si) => si !== setImp);
   if (classMappingsWithSimilarTarget.length) {
-    setImp.setRoot(false);
```

Comment:
looking at these, I wonder if these methods should be here or `legend-studio`

Reply:
In particular, as I just checked, let's move
`updateRootSetImplementationOnCreate`, `updateRootSetImplementationOnDelete`, and `nominateRootSetImplementation` to `legend-studio` and leave the rest of them in `legend-graph`, also it's safe to move them to say `MappingHelper`. Also, we could eliminate `classMappingsWithSimilarTarget[0].setRoot(true);` I don't even trust the logic we have in `findRootSetImplementation` anymore.